.github/workflows/qa.yaml

```diff
@@ -9,14 +9,48 @@ on:
       - "renovate/*"
   pull_request:
 
+# Cancel existing jobs for this branch/PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+
 jobs:
   lint:
     uses: canonical/starflow/.github/workflows/lint-python.yaml@main
   test:
-    uses: canonical/starflow/.github/workflows/test-python.yaml@main
+    uses: canonical/starflow/.github/workflows/test-python.yaml@work/setup-vars
     with:
       fast-test-platforms: '["ubuntu-22.04", "ubuntu-24.04"]'
```

Comment:
issue: that is probably unrelated to this PR but I noticed that tests on plugins are still executed in "fast tests" jobs, see https://github.com/canonical/craft-parts/actions/runs/17602626438/job/50008521822?pr=1275#step:4:782. Unless I am missing something, I suspect passed markers are not respected, or not properly passed to the `make test-coverage` command.